### DISCUSSION TITLE
Explanation required on using config.xml

### DISCUSSION THREAD

**Discussion by nitishfy** [checkstyle/checkstyle#12459] [Q&A] [answered]
I'm currently working on this [issue](https://github.com/checkstyle/checkstyle/issues/7598) which requires adding specific examples.

I've created the `config.xml` in a local folder along with the following contents and a jar file (you can see it's present there).
![image](https://user-images.githubusercontent.com/86051118/204076794-9d587821-755a-45b7-b17b-297e1a5387fb.png)

Can anyone help me with how to use this config.xml (what command is required to run only this check present in config.xml?)

As you can see, There's jar file as well. Running checkstyle with jar will run all the checks on a simple java code. Is there any way possible to run a check for addressing only JavaDocMethod in order to solve the issue mentioned above?

**Comment by rnveach** [comment, marked as answer]
https://checkstyle.org/cmdline.html#Command_line_usage
https://checkstyle.org/cmdline.html#Download_and_Run

> Running checkstyle with jar will run all the checks on a simple java code

This is not true. There is no run all checks option in checkstyle. We run what you define in the config you wish to use against the files you choose.

**Reply by nitishfy** [reply]
Thank you!
